OSDOCS-14694 created 4.14.52 z-stream release notes (#93548)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

// 4.14.52
[id="ocp-4-14-52_{context}"]
=== RHSA-2025:7702 - {product-title} 4.14.52 bug fix and security update

Issued: 21 May 2025

{product-title} release 4.14.52, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:7702[RHSA-2025:7702] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:7704[RHBA-2025:7704] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.52 --pullspecs
----

[id="ocp-4-14-52-bug-fixes_{context}"]
==== Bug fixes

> * Previously, an operator would incorrectly appear twice in the installed operators list if the current project matched its default namespace and the copied CSVs were disabled in {olm-first}. With this release, the operator displays only once. (link:https://issues.redhat.com/browse/OCPBUGS-55942[OCPBUGS-55942])

Reviewer: ```suggestion
* Previously, an operator would incorrectly appear twice in the installed operators list if the current project matched its default namespace and the copied CSVs were disabled in {olmv1-first}. With this release, the operator displays only once. (link:https://issues.redhat.com/browse/OCPBUGS-55942[OCPBUGS-55942])
```
That attribute doesn't exist in the 4.14 branch; @michaelryanpeter can we use the v1.0 attribute in this case, or should we just say "Operator Lifecycle Manager" and call it done?

Reviewer 2: Correct. I would switch to "Operator Lifecycle Manager." Also, Operator needs to be capitalized throughout.

> * Previously, an operator would incorrectly appear twice in the installed operators list if the current project matched its default namespace and the copied CSVs were disabled in Operator Lifecycle Manager (OLM). With this release, the operator displays only once. (link:https://issues.redhat.com/browse/OCPBUGS-55942[OCPBUGS-55942])

Reviewer 3: ```suggestion
* Previously, an operator would incorrectly appear twice in the installed operators list if the current project matched its default namespace and the copied CSVs were disabled in {olm-first}. With this release, the operator displays only once. (link:https://issues.redhat.com/browse/OCPBUGS-55942[OCPBUGS-55942])
```
Swapping out the hard-coded name for the attribute.

Author: Added attribute

Author: Change made

> * Previously, {product-title} release 4.14.52 and later releases remained on the stable-4.14 channel instead of switching to the eus-4.14 channel. This caused 'VersionNotFound' errors when attempting to retrieve update information and since the version is not found in the stable-4.14 channel, the recommended updates are not available. With this release, the installer directs 4.14.52 and later releases to the Extended Update Support (EUS) channel ensuring the releases receive recommended update notices and the `VersonNotFound` error no longer displays. (link:https://issues.redhat.com/browse/OCPBUGS-55193[OCPBUGS-55193])

Reviewer 3: ```suggestion
* Previously, {product-title} release 4.14.52 and later releases remained on the stable-4.14 channel instead of switching to the eus-4.14 channel. This caused 'VersionNotFound' errors when attempting to retrieve update information. Because the version was not found in the stable-4.14 channel, the recommended updates were not available. With this release, the installer directs 4.14.52 and later releases to the Extended Update Support (EUS) channel, ensuring the releases receives the recommended update notices, and the `VersonNotFound` error no longer displays. (link:https://issues.redhat.com/browse/OCPBUGS-55193[OCPBUGS-55193])
```

Author: Changes made

* Previously, when the `openshift-host-network` namespace was modified by the user or by an upgrade, the network policy did not correctly set the VXLAN virtual network ID parameter, `VNID`, to `0`. With this release, the VNID parameter is correctly set after the namespace is modified. (link:https://issues.redhat.com/browse/OCPBUGS-54868[OCPBUGS-54868])

> * Previously, cluster nodes repeatedly lose communication due to improper remote port binding by Open Virtual Network (OVN)-Kubernetes. This affects pod communication across nodes. With this release, the remote port binding functionality has been updated to be handled by OVN directly, improving the reliability of cluster node communication. (link:https://issues.redhat.com/browse/OCPBUGS-48522[OCPBUGS-48522])

Reviewer 3: ```suggestion
* Previously, cluster nodes repeatedly lost communication due to improper remote port binding by Open Virtual Network (OVN)-Kubernetes. This affected pod communication across nodes. With this release, the remote port binding functionality has been updated to be handled by OVN directly, improving the reliability of cluster node communication. (link:https://issues.redhat.com/browse/OCPBUGS-48522[OCPBUGS-48522])
```

Author: Changes made

> * Previously, when installing a cluster on {ibm-cloud-name} into an existing Virtual Private Cloud (VPC), the installation program retrieved an unsupported VPC region. Attempting to install into a supported VPC region that follows the unsupported VPC region alphabetically caused the installation program to crash. With this release, the installation program ignores any VPC regions that are not fully available during resource lookups. (link:https://issues.redhat.com/browse/OCPBUGS-48196[OCPBUGS-48196])

Reviewer 3: ```suggestion
* Previously, when you installed a cluster on {ibm-cloud-name} into an existing Virtual Private Cloud (VPC), the installation program retrieved an unsupported VPC region. If you tried to install into a supported VPC region that followed the unsupported VPC region alphabetically, the installation program crashed. With this release, the installation program ignores any VPC regions that are not fully available during resource lookups. (link:https://issues.redhat.com/browse/OCPBUGS-48196[OCPBUGS-48196])
```

Author: Changes made
